api/src/main/resources/Covid-19_Concepts-1.xml

```diff
@@ -0,0 +1,463 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<dataset>
+<concept_datatype concept_datatype_id="1" name="Numeric" hl7_abbreviation="NM" description="Numeric value, including integer or float (e.g., creatinine, weight)" creator="1" date_created="2004-02-02 00:00:00.0" retired="false" retired_by="1" uuid="8d4a4488-c2cc-11de-8d13-0010c6dffd0f"/>
+<concept_datatype concept_datatype_id="2" name="Coded" hl7_abbreviation="CWE" description="Value determined by term dictionary lookup (i.e., term identifier)" creator="1" date_created="2004-02-02 00:00:00.0" retired="false" retired_by="1" uuid="8d4a48b6-c2cc-11de-8d13-0010c6dffd0f"/>
```

Comment:
Are these new concept data types that never existed before?

Reply:
@dkayiwa , i have removed all the existing concept data types and concept clases from this new package